command/commands.go

```diff
@@ -82,6 +82,11 @@ const (
 	EnvVaultLicensePath = "VAULT_LICENSE_PATH"
 	// EnvVaultDetailed is to output detailed information (e.g., ListResponseWithInfo).
 	EnvVaultDetailed = `VAULT_DETAILED`
+	// EnvVaultLogFile is used to specify the path to the log file that Vault should use for logging
+	EnvVaultLogFile = "VAULT_LOG_FILE"
```

Comment:
Given that this environment variable is Agent-specific, it might makes sense to denote that in the comment above.

Reply:
You're right @ccapurso, however, it's soon going to be for agent + server so it might be worth just leaving it as is for now though?

Reply:
Oh great! I think I missed that context, whoops.